bokehjs/src/lib/models/glyphs/step.ts

```diff
@@ -95,9 +95,9 @@ export namespace Step {
     mode: p.Property<StepMode>
   } & Mixins
 
-  export type Mixins = LineVector
+  export type Mixins = Line/*Vector*/
 
-  export type Visuals = XYGlyph.Visuals & {line: Line}
+  export type Visuals = XYGlyph.Visuals & {line: visuals.Line/*Vector*/}
```

Comment:
@bryevdv , `Step` glyph defines line visuals as vectorized both in bokeh and bokehjs, but the implementation uses `this.visuals.line.set_value(ctx)`. I went with the implementation, but I'm not sure which is correct.

Reply:
Step is like `Line`, there's a vector of points, but the connected topology means there should be only one color (etc) for the entire thing. Sos, should be scalar IMO